Update CosmWasm SECURITY.md (#4)
CosmWasm/advisories: SECURITY.md

- Once a security report is received, the core development team works to verify the issue.
- Patches are prepared for eligible releases in private repositories.
- We notify the community that a security release is coming, to give users time to prepare their systems for the update.
  - The notification contains the release date and time (72-24 hours after the notification).

>   - The notification contains as little information as possible to make it hard for attackers to guess where to search. However, depending on the situation it may contain additional information such as whether or not the patch will be consensus breaking or chains with permissioned CosmWasm integration are equally affected as permissionless chains.

Reviewer: ```suggestion
  - The notification contains as little information as possible, to make it hard for attackers to guess where to search. However, depending on the situation it may contain additional information such as whether the patch will be consensus breaking, or if chains with permissioned CosmWasm integration are equally affected as permissionless chains.
```

>   - The projects on the [notification list](https://github.com/CosmWasm/advisories#notification-list) is informed first. Please add yourself via a PR if you want to receive notifications.

Reviewer: ```suggestion
  - The projects on the [notification list](https://github.com/CosmWasm/advisories#notification-list) are informed first. Please add yourself via a PR if you want to receive notifications.
```

>   - Right after that the notification is broadcasted publically including Discord messages, tweets, and emails to partners and validators.

Reviewer: ```suggestion
  - Right after that the notification is publicly broadcast, including Discord messages, tweets, and emails to partners and validators.
```
